checkov/terraform/variable_rendering/safe_eval_functions.py

```diff
@@ -176,3 +176,6 @@ def wrap_func(f, *args):
 SAFE_EVAL_DICT['tonumber'] = lambda arg: arg if type(arg) in [int, float] else wrap_func(tonumber, arg)
 SAFE_EVAL_DICT['toset'] = lambda origin: set(origin)
 SAFE_EVAL_DICT['tostring'] = lambda arg: arg if isinstance(arg, str) else wrap_func(tostring, str(arg))
+
+# encoding
+SAFE_EVAL_DICT['jsonencode'] = lambda arg: arg
```

Comment:
interesting. we had some challenges around encoding in the past. @nimrodkor @rotemavni @tronxd can I have another set of eyes and security guidelines on those? just to see we are not missing anything

Reply:
@schosterbarak Yep it's good :)